### DISCUSSION TITLE
merge_topic : how to identify if new clusters come from merge_topic list

### DISCUSSION THREAD

**Discussion by alexcpop** [MaartenGr/BERTopic#953] [Q&A]
Hi everyone, 

I have a list of around 170 clusters, some of which are similar and so I decided to use 'merge_topic' to merge them together and decrease the clusters size. It's unclear how some of the new merged clusters relate to the 'merge_topic' list. Is there a way to check say the new cluster 0 , relates to the second list in the 'merge_topic'? Thank you

I used this as an example:
topics_to_merge = [[1, 2],
                   [3, 4]]
topic_model.merge_topics(docs, topics_to_merge)

**Comment by MaartenGr** [comment]
The merged topics are indeed not directly tracked between operations as that would require quite some tracking. Instead, you can take a document from topic 1 before merging and see what topic it becomes after merging. The updated topics can be found in `topic_model.topics_`.
